doc/source/examples/dem/rotating-drum/rotating-drum.rst

```diff
@@ -167,14 +166,14 @@ This simulation can be launched in two steps. First the particles need to be loa
 
   mpirun -np 8 dem load-rotating-drum.prm
 
-Then the rotation of the drum is started in a second simulation:
+Then we run the simulation with the rotating walls:
 
 .. code-block:: text
 
-  mpirun -np 8 dem load-rotating-drum.prm
+  mpirun -np 8 dem rotating-drum.prm
 
 .. warning::
-	Loading the particles in this example requires approximately 50 minutes on 8 cores. Simulating the motion of the particles requires an additional 8 hours. This high computational cost is because of the large number of particles and the long duration of the simulation.
+	This example needs a simulation time of approximately 12 hours 8 cores. 
```

Comment:
```suggestion
	This example needs a simulation time of approximately 12 hours on 8 cores. 
```

Reply:
That's mistake. I actually commited the two new prm files when I did the refactoring of the dim of the solver. So it confused the history. That's why the prm are missing from the commits also.